For the CI test that failed, here is the error message:
```
>> 1015    /tmp/tmp-mount-MGMMwD/diskspace_workaround_spack_stack/spack-cache
             /build_stage/spack-stage-ecflow-5.8.4-3pmfndgkeqw674p7zpbvnlzep7b2
             psio/spack-build-3pmfndg/Viewer/libViewer/src/moc_LocalSocketServe
             r.cpp:10:10: fatal error: cannot open file '/tmp/tmp-mount-MGMMwD/
             diskspace_workaround_spack_stack/spack-cache/build_stage/spack-sta
             ge-ecflow-5.8.4-3pmfndgkeqw674p7zpbvnlzep7b2psio/spack-build-3pmfn
             dg/Viewer/libViewer/src/../../../../../../../../../../Users/ec2-us
             er/spack-stack/spack-cache/build_stage/spack-stage-ecflow-5.8.4-3p
             mfndgkeqw674p7zpbvnlzep7b2psio/spack-src/Viewer/libViewer/src/Loca
             lSocketServer.hpp': Permission denied
     1016    #include "../../../../../../../../../../Users/ec2-user/spack-stack
             /spack-cache/build_stage/spack-stage-ecflow-5.8.4-3pmfndgkeqw674p7
             zpbvnlzep7b2psio/spack-src/Viewer/libViewer/src/LocalSocketServer.
             hpp"
     1017             ^
     1018    1 error generated.
  >> 1019    make[2]: *** [Viewer/libViewer/src/CMakeFiles/viewer.dir/moc_Local
             SocketServer.cpp.o] Error 1
     1020    make[2]: *** Waiting for unfinished jobs....
```

Looks like the build of ecflow failed. Any ideas what to do to fix this? Or should I simply re-run the tests? Thanks!

--- a/.github/workflows/macos-ci-aarch64.yaml
+++ b/.github/workflows/macos-ci-aarch64.yaml
@@ -112,10 +112,10 @@ jobs:
           spack buildcache list
 
           # Workaround for limited disk space on macOS arm instance
-          spack config add "config:build_stage:/Users/ec2-user/spack-stack/spack-cache/build_stage"
-          spack config add "config:test_stage:/Users/ec2-user/spack-stack/spack-cache/test_stage"
-          spack config add "config:source_cache:/Users/ec2-user/spack-stack/spack-cache/source_cache"
-          spack config add "config:misc_cache:/Users/ec2-user/spack-stack/spack-cache/misc_cache"
+          spack config add "config:build_stage:/tmp/tmp-mount-MGMMwD/diskspace_workaround_spack_stack/build_stage"
+          spack config add "config:test_stage:/tmp/tmp-mount-MGMMwD/diskspace_workaround_spack_stack/test_stage"
+          spack config add "config:source_cache:/tmp/tmp-mount-MGMMwD/diskspace_workaround_spack_stack/source_cache"
+          spack config add "config:misc_cache:/tmp/tmp-mount-MGMMwD/diskspace_workaround_spack_stack/misc_cache"
 
           # Break installation up in pieces and create build caches in between
           # This allows us to "spin up" builds that altogether take longer than

--- a/.gitmodules
+++ b/.gitmodules
@@ -2,10 +2,8 @@
   path = spack
   #url = https://github.com/spack/spack
   #branch = develop
-  #url = https://github.com/jcsda/spack
-  #branch = jcsda_emc_spack_stack
-  url = https://github.com/srherbener/spack
-  branch = feature/openblas-0.3.24
+  url = https://github.com/jcsda/spack
+  branch = jcsda_emc_spack_stack
 [submodule "doc/CMakeModules"]
   path = doc/CMakeModules
   url = https://github.com/noaa-emc/cmakemodules

